image/templates/helm/shared/templates/02-scanner-v4-01-security.yaml

```diff
@@ -0,0 +1,79 @@
+{{- include "srox.init" . -}}
+
+{{- if or ._rox.scannerV4._indexerEnabled ._rox.scannerV4._matcherEnabled }}
+{{- if and ._rox.env.openshift ._rox.system.createSCCs }}
+kind: SecurityContextConstraints
```

Comment:
Could we stick to the SCCs that OpenShift 4 ships out of the box if possible?
I am working on a [PR right now](https://github.com/stackrox/stackrox/pull/8663) to deprecate all custom SCCs (ROX-12982), and it would be a lot of duplicated work if we were to introduce new ones to only deprecate them straight afterwards.
The current Scanner is able to run under `nonroot-v2`. It could also run as `restricted-v2` if we could deprecate the fixed `uid:gid` that it is running as. I would expect Scanner V4 to behave very similar.